lib/store_web/live/backoffice/order_live/show.ex

```diff
@@ -0,0 +1,69 @@
+defmodule StoreWeb.Backoffice.OrderLive.Show do
+  use StoreWeb, :live_view
+
+  import Ecto.UUID
+  import Ecto.Query
+  alias Store.Repo
+  alias Store.Inventory
+  alias Store.Inventory.Order
+  alias Store.Inventory.OrdersProducts
+  alias Store.Uploaders
+  alias Store.Accounts
+
+  @impl true
+  def mount(%{"id" => id}, _session, socket) do
+    {:ok, assign(socket, order: Inventory.get_order!(id))}
+  end
+
+  @impl true
+  def handle_params(%{"id" => id}, _, socket) do
+    {:noreply,
+     socket
+     |> assign(:current_page, :orders)
+     |> assign(:page_title, page_title(socket.assigns.live_action))
+     |> assign(:order, Inventory.get_order!(id) |> Repo.preload(:products))}
+  end
+
+  @impl true
+  def handle_event("paid", _payload, socket) do
+    order = socket.assigns.order
+
+    order
+    |> Order.changeset(%{status: :paid})
+    |> Repo.update!()
+
+    {:noreply, socket}
+
+  end
+
+
+  @impl true
+  def handle_event("delivered", _payload, socket) do
+    order = socket.assigns.order
+
+    order
+    |> Order.changeset(%{status: :delivered})
+    |> Repo.update!()
+
+    {:noreply, socket}
+
+  end
+
+
+  defp capitalize_status(status) do
+    status
+    |> Atom.to_string()
+    |> String.capitalize()
+  end
+
+  defp total_price(order) do
+    Enum.reduce(order.products, 0, fn product, acc -> acc + product.price end)
+  end
+
+  defp user_email(id) do
+    Accounts.get_user!(id).email
+  end
```

Comment:
You are using this functions in both index and show. Can't you just create them in `Inventory.ex`, for example? There has to be a better approach 💪🏼